meerk40t/gui/wxmmain.py

```diff
@@ -1828,15 +1850,14 @@ def on_focus_lost(self, event):
         # event.Skip()
 
     def on_click_new(self, event=None):  # wxGlade: MeerK40t.<event_handler>
-        context = self.context
-        self.working_file = None
-        self.validate_save()
-        context.elements.clear_all()
-        self.context(".laserpath_clear\n")
```

Comment:
I figured it was a good idea to extract this out so that I could re-use the function in clear_and_open.